### ISSUE TITLE
[Bug] Bitbucket build created when deleted branch

### ISSUE THREAD

**Issue by garas** [php-censor/php-censor#337]
## Expected behavior

PHP Censor should ignore webhook after deleting branch if `"new": null`

## Actual behavior

PHP Censor creates build for default branch

## Steps to reproduce

1. Delete branch from Bitbucket
2. Webhook received:
```json
{
  "push": {
    "changes": [
      {
        "forced": false,
        "old": {
          "name": "deletedbranch",
          "type": "branch",
        },
        "created": false,
        "truncated": false,
        "closed": true,
        "new": null
      }
    ]
  } 
}
```
Note `"new": null`

PHP Censor response:
```json
{
  "status": "ok",
  "commits": {
    "": {
      "status": "ok",
      "buildID": 1409
    }
  }
}
```

## Environment

* PHP Censor version 1.1.1 
* Operating System CentOS 6.10
* PHP version 7.2.24
* MySQL version 10.2.29-MariaDB-log

**Comment by corpsee** [comment]
@garas Thanks!

**Comment by corpsee** [comment]
@garas Could you test your case in `bitbucket-fix` branch please?

**Comment by garas** [comment]
No build created ✔ Thanks for fix.

PHP Censor response was now HTTP status: 200
```json
{
  "status": "failed",
  "commits": []
}
```

**Comment by corpsee** [comment]
@garas Thank you for testing! Changes merged into branches `release-1.0`, `release-1.1` and `master` and will be released with versions `1.0.13`, `1.1.3` and `1.2.0` (In the future).
